src/extension/NUnit2FrameworkDriver.cs

```diff
@@ -66,16 +64,37 @@ public NUnit2FrameworkDriver(AppDomain testDomain)
 
             var initializer = DomainInitializer.CreateInstance(_testDomain);
             initializer.InitializeDomain((int)InternalTrace.Level);
-
-            _runner = RemoteTestRunner.CreateInstance(_testDomain, 1);
         }
 
         public string ID { get; set; }
 
+        private TestRunner Runner {
+            get
+            {
+                if (_runner == null)
+                {
+                    int runnerId;
+                    if (ID == null)
+                    {
+                        throw new NUnitEngineException("NUnit 2 Driver requires that ID must be defined at first.");
+                    }
```

Comment:
I'm not sure about this test. In fact, the driver requires no such thing. It's very happy with a null id. However, any runner that wants to use multiple instances of the driver will typically need to set it.

@rprouse What do you think is the right thing to do here? How will our decision affect future drivers that may be written? (Assuming we establish this as a general precedent)